docs/Controllers.md

```diff
@@ -0,0 +1,87 @@
+# Controllers
+
+_(c) AMWA 2022, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
+
+## Informative
+
+A Controller is Client software that interacts with the NMOS APIs to discover, connect and manage devices within a networked media system.
+
+* This document includes normative references to be followed when implementing a Controller.
+* This document covers how the Controller interacts with the NMOS APIs only.
+It does not cover other features of the Controller software, such as presentation.
+* This document does not cover any requirements relating to where a Controller is additionally acting as a Node (e.g. receiving monitoring information via IS-07).
+
+## General
+
+### User
+Where this document refers to the "user" of a Controller, this includes both human operators who drive the Controller manually and automation systems that drive the Controller programmatically.
+
+### HTTP APIs
+#### Trailing Slashes
+APIs may advertise URLs with or without a trailing slash.
```

Comment:
My personal preference would be for more vertical whitespace in Markdown...

```suggestion
### HTTP APIs

#### Trailing Slashes

APIs may advertise URLs with or without a trailing slash.
```

Reply:
Please consider changing lowercase `may` to something that can't be confused with conformance language.

Reply:
Noted and agreed. For reference this sentence was lifted from the [AMWA Wiki](https://github.com/AMWA-TV/nmos/wiki/Common-Issues#http-apis) for future updates.